dataduct/steps/load_redshift.py

```diff
@@ -17,6 +17,7 @@ def __init__(self,
                  insert_mode="TRUNCATE",
                  max_errors=None,
                  replace_invalid_char=None,
+                 compression=None,
```

Comment:
Can you please edit the steps documentation here: https://github.com/coursera/dataduct/blob/develop/docs/steps.rst with the new option as well. (Under the load redshift section.)